### DISCUSSION TITLE
Distinguish Between Lists and Starred Projects

### DISCUSSION THREAD

**Discussion by chaoscommencer** [community/community#49868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I feel like lists and starred projects should be handled differently.  For example, just because I group a project in a list, I do not automatically want it to be starred.  A starred project I would assume to be a favorite--something I would want to visit regularly should be starred.

The issue is that right now, there is no distinction.  As soon as a project is placed in a list/group, it is automatically starred.  Regardless of whether you frequent the project or really like it, the repository cannot--in the current state--be un-starred without also removing it from any lists.

This is unintuitive UX behavior, and it destroys the usefulness of a star feature.  For someone who likes to be organized, everything will end up in at least one list, and so everything will be starred.

Others have similar complaints (although I feel the above includes a more complete description):
- https://github.com/community/community/discussions/39549
- https://github.com/community/community/discussions/41458
- https://github.com/community/community/discussions/38192
- https://github.com/community/community/discussions/10272
- https://github.com/community/community/discussions/9315
- https://github.com/community/community/discussions/37393
- https://github.com/community/community/discussions/8663
- https://github.com/community/community/discussions/10262
- https://github.com/community/community/discussions/16989
- https://github.com/community/community/discussions/19638
- https://github.com/community/community/discussions/8702
- https://github.com/community/community/discussions/9181
- https://github.com/community/community/discussions/8842
- https://github.com/community/community/discussions/11041
Counter-arguments (claimed to be a bug):
- https://github.com/community/community/discussions/41841

Included some confusing UX tickets in the 'for' list.

Additionally, it's apparent there are several bugs that have been spawned out of trying to unite these two features.  Those bugs would be avoided by keeping them separate as proposed.

**Comment by abas929** [comment]
ال شي لا
